client/blocks/reader-import-button/index.jsx

```diff
@@ -57,18 +55,40 @@ class ReaderImportButton extends React.Component {
 		} );
 
 		if ( err ) {
-			this.props.onError( err );
+			this.onImportFailure( err );
 		} else {
 			// Tack on the file name since it will be displayed in the UI
 			data.fileName = this.fileName;
-			this.props.onImport( data );
+			this.onImportSuccess( data );
 		}
 	}
 
 	onImportProgress = ( event ) => {
 		this.props.onProgress( event );
 	}
 
+	onImportSuccess = ( feedImport ) => {
+		const message = this.props.translate(
+			'{{em}}%(name)s{{/em}} has been received. You\'ll get an email when your import is complete.',
+			{
+				args: { name: feedImport.fileName },
+				components: { em: <em /> },
+			}
+		);
+		this.props.successNotice( message );
+	}
+
+	onImportFailure = ( error ) => {
+		if ( ! error ) {
+			return null;
+		}
+
+		const message = this.props.translate( 'Whoops, something went wrong. %(message)s Please try again.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Whoops, something went wrong. %(message)s. Please try again.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).